packages/analytics-js/src/services/ErrorHandler/ErrorHandler.ts

```diff
@@ -180,16 +204,18 @@ class ErrorHandler implements IErrorHandler {
    *
    * @param {Error} error Error instance from handled error
    */
-  notifyError(error: Error) {
-    if (this.errReportingClient && isAllowedToBeNotified(error)) {
+  notifyError(error: SDKError, errorState: ErrorState) {
+    if (this.pluginEngine && isAllowedToBeNotified(error)) {
```

Comment:
```suggestion
    if ((this.httpClient || this.pluginEngine) && isAllowedToBeNotified(error)) {
```

Reply:
Yes. In fact, both are required to support both old and new plugin versions. Core SDK does not know which plugins it is interacting with.

Reply:
Any change needed here?

Reply:
Yes. `(this.httpClient && this.pluginEngine)`

Reply:
When invoking old plugin this.httpClient is not needed

Reply:
The core SDK does not have a clue on what version of the plugins set it is interacting with.
So, from the core SDK perspective, both of them should be treated as mandatory.

Reply:
done